doc/language-reference.rst

```diff
@@ -475,6 +475,27 @@ instances of *CLS* whose attribute *Number* is larger than 100.
 
     .select many inst_set from instances of CLS where (selected.Number > 100)
 
+Ordering Selections
+-------------------
+Instance selections returned by *select many* statement may be ordered by one or
+more instance attributes. The *ordered_by* keyword sorts the resulting set in
+ascending order and the *reverse_ordered_by* keyword sorts the resulting set in
+descending order. If multiple attributes are specified, the set will be sorted
+by the first attribute and then within each value of this, by the second
+attribute and so on. The following example demonstrates how to select all the
+instances of *PERSON* and order them by age first, then alphanumerically by name.
+
+.. code-block:: pyrsl
+
+    .select many people from instances of PERSON ordered_by ( age, name )
```

Comment:
I don't like the superfluous spaces within a group of parenthesis, i.e. ordered_by ( age, name ) --> ordered_by (age, name).

Reply:
done